internal/otelcollector/config/log/agent/receivers.go

```diff
@@ -0,0 +1,94 @@
+package agent
+
+import (
+	"fmt"
+
+	"k8s.io/utils/ptr"
+)
+
+func makeReceivers() Receivers {
+	return Receivers{
+		FileLog: &FileLog{
+			Exclude: []string{
+				"/var/log/pods/kyma-system_telemetry-log-agent*/*/*.log",
+				"/var/log/pods/kyma-system_telemetry-fluent-bit*/*/*.log",
```

Comment:
these names highly depend on the namespace the telemetry module is using and the name of the daemonset. imho we should make this configurable